README.md

```diff
@@ -1,5 +1,17 @@
-> Name to be found
-
-# PyFlag Bot
+# RootPythia
 
 **TODO: everything**
+
+### API Root-me
+
+**Info about the API**
+
+>A firewall makes us safe against Deny of Service attacks, banishing every IP address that :
+>- initiates more than 25 connections per second
+>- maintains more than 25 TCP connections simultaneously
+
+>This banishment is temporary and lasts only 5 minutes. Don’t try to connect to our services during ban time or it will be extended.
+
+### PNG creator
+
+Create an image to show the new validated challenge
```

Comment:
I quite don't understand how your README commit ended up in both PRs :laughing: but this is fine I think, you'll probably have a confloicts or a discarded commit once one of the two will be merged though